service/src/main/java/org/mskcc/cmo/metadb/service/impl/MessageHandlingServiceImpl.java

```diff
@@ -178,12 +158,6 @@ public void onMessage(Message msg, Object message) {
                     messageHandlingService.newRequestHandler(metaDbRequest);
                 } catch (Exception e) {
                     LOG.error("Exception during processing of request on topic: " + IGO_NEW_REQUEST_TOPIC, e);
-                    try {
-                        requestStatusLogger.logRequestStatus(message.toString(),
-                                RequestStatusLogger.StatusType.REQUEST_PARSING_ERROR);
```

Comment:
Shouldn't we log parsing errors

Reply:
those would be handled in the request filter app - I have a separate PR w/some changes to that

Reply:
@divyamadala30 here it is https://github.com/mskcc/cmo-metadb-request-filter/pull/6/files

Reply:
Doesn't this catch errors while converting the message into a java object?